src/platform.ts

```diff
@@ -30,12 +32,12 @@ export class BlueAirHomebridgePlatform implements DynamicPlatformPlugin {
     // initializing login information
     this.log = log;
 
-    if(config.username == undefined || config.password == undefined){
+    if(config.username === undefined || config.password === undefined){
       this.log.error('Missing BlueAir API credentials.');
       return;
     }
 
-    this.blueair = new BlueAirApi(this.log, config.username, config.password);
```

Comment:
won't removal of this line prevent the old blueair API from being properly initialized?

Reply:
Very much so! I just had a fun 15 minutes where I was staring at the code trying to figure out why I was getting a TypeError. Latest commit should not only fix that, but finish the refactor.